src/lib/mina_base/zkapp_state.ml

```diff
@@ -1,10 +1,10 @@
 open Core_kernel
 open Pickles_types
-module Max_state_size = Nat.N8
+module Max_state_size = Nat.N32
 
 module State_length_vec :
   Vector.VECTOR with type 'a t = ('a, Max_state_size.n) Vector.vec =
-  Vector.Vector_8
+  Vector.Vector_32
```

Comment:
A little down from here in the code, this is used in a stable type:

https://github.com/MinaProtocol/mina/pull/17462/files#diff-13a1f5b90728433057d97d01b042224062d0fba9a101eae1830cfd784813d4c2R21

So that's going to have to be bumped at some point, as well as the versions of stable types that use this in their definition (transitively).